### DISCUSSION TITLE
Using Container only on Linux

### DISCUSSION THREAD

**Discussion by pmatos** [community/community#25524] [Actions]
<p>I have a matrix job that uses both ubuntu-18.04 and macos. But on Linux I don’t want to use the VM directly and would prefer to use a container instead.</p>
<p>Is there a way to specify a container only if the matrix is using ubuntu and not on macos?</p>
<p>Paulo</p>

**Comment by dawidd6** [comment, marked as answer]
<p>Similar topic: <a href="https://github.community/t/run-matrix-job-on-macos-and-on-ubuntu-in-container/16359/3">https://github.community/t5/GitHub-Actions/Run-matrix-job-on-macos-and-on-ubuntu-in-container/m-p/37013#M2772</a></p>

**Comment by pmatos** [comment]
<p>Thanks - marking your reference as solution since this is a duplicated issue.</p>
